README.md

```diff
@@ -39,6 +39,17 @@ Refer to [the documentation](https://hexdocs.pm/bridge_ex/BridgeEx.Graphql.html)
 
 If you need more control on your requests you can use [`BridgeEx.Graphql.Client.call`](https://hexdocs.pm/bridge_ex/BridgeEx.Graphql.Client.html#call/7) directly.
 
+The library supports preloading queries from external files via the `BridgeEx.Extensions.ExternalResources` optional macro:
```

Comment:
Maybe this should be moved under `## Extensions`? Aside from this I'd say this PR looks good 😊

Reply:
Oh sorry, I initially added this to the "extensions" section, but then I moved them up to give them more visibility. Do you prefer a dedicated section?

Reply:
Right now we have just one extension so I'd say it makes sense to leave it where it is and just delete the section 👍